ReadMe.md

```diff
@@ -51,10 +53,31 @@ setTimeoutAt(function (msg) {
 }, Date.now() + 5000, 'Hello world');
 ```
 
+To cancel a `Timeout`, use the instance's `clear` method, or pass it as an argument to `clearTimeout`:
```

Comment:
I feel like there still needs to be some sort of heading (`**...**` like above) right before this line to help separate it from the `setTimeoutAt` method description, _especially_ since `Timeout` in particular has two creation methods.